pandas/tests/groupby/test_function.py

```diff
@@ -262,6 +264,64 @@ def _check(self, df, method, expected_columns, expected_columns_numeric):
 
         tm.assert_index_equal(result.columns, expected_columns)
 
+    def test_groupby_aggregation_non_numeric_dtype(self):
```

Comment:
move near this tests: 

pandas/tests/groupby/test_groupby.py:def test_groupby_empty_multi_column(as_index, numeric_only):


OT we should move the tests of numeric_only=False to a separate location

Reply:
> move near this tests:
> 
> pandas/tests/groupby/test_groupby.py:def test_groupby_empty_multi_column(as_index, numeric_only):
> 
I've done this in https://github.com/pandas-dev/pandas/pull/43154/commits/45f54d6cb548828072fc2d5885687d4d4db5b712